### ISSUE TITLE
Should react-dnd try to overcome Parcel bundling issues?

### ISSUE THREAD

**Issue by youknowriad** [react-dnd/react-dnd#945]
Related discussion https://twitter.com/riadbenguella/status/949932743810535424

React DnD is not usable right now with the parcel bundler. This is not an issue with "react-dnd" but more about how Parcel handles/transpiles dependencies. If it finds a `.babelrc` file on the dependency, it tries to transpile the said package using this config.

React DnD (in parcel) suffers from this issue due to the disposable package here https://github.com/gaearon/disposables/pull/7 

I just wanted to ask if you think it's a good idea to try to overcome this by ignoring `.babelrc` while publishing the modules or should we just wait on a decision from the parcel bundler maintainers. Discussion here https://github.com/parcel-bundler/parcel/issues/13

Thanks

**Comment by gaearon** [comment]
I wanted to transfer `disposables` to this org but forgot my password and github locked me out. So I just went ahead and published `disposables@1.0.2` without `.babelrc`.

I wish we did a better job educating people in the issues about this instead of treating it as a real problem. Just explaining that you're not supposed to be compiling `node_modules` with Babel would probably be a good idea and save some confusing in those threads. :-) I understand if the current maintainers didn't have time for this (maybe somebody wants to help?)
